.npmignore

```diff
@@ -1,2 +1,3 @@
 .git*
 test/
+docs
```

Comment:
This will prevent publishing the `docs` directory to npm which is usually not advised.